backend/src/main/java/com/ideahub/model/IdeaCollaboratorId.java

```diff
@@ -1,57 +1,25 @@
 package com.ideahub.model;
```

Comment:
This one is responsible for the composite key, and I just noticed it's missing one annotation (@Embeddable)